### DISCUSSION TITLE
part-ii/2017-6/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#59] [Question comments]
Part II 2017 Algebraic Topology: Paper 1, Section II, I 

https://questions.tripos.org/part-ii/2017-6/

**Comment by eha1234** [comment]
I thought this was quite fun, but hard to write down, and going by the rather scathing examiners comment, I think other people found this to. 

<details><summary>[Spoilers]</summary>

(a), (b) bookwork 
(c) The element conjugating you get from $H(t, x_0)$ 
(d) I lifted to $\mathbb{R}^2$ but I also feel like this isn't what they intended?
(e) $a, bab^{-1}$, where $a, b$ are the loops generating the fundamental group. 

</details>

Comments and corrections appreciated!

**Reply by dropdbassett** [reply]
I did something similar to you for part (d)! Everyone I've spoken to about it also has so I feel like this somehow was what was intended.

I had the following thoughts about part (e):
<details><summary>Part (e)</summary>

I'm not sure what the homotopy was for $a$ and $bab^{-1}$, but I chose $ab$ and $ba$ - note that we have $ab = ba \circ A$, where $A$ is the antipodal map on $S^1$. But this is a rotation by $\pi$, so by considering rotation by $\pi t$ it is homotopic to the identity map on $S^1$.
</details>
